frontend/src/config/shared/arabic.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
You can also use this for jordan and remove the jordan/translation.json as part of this PR

Reply:
done